src/HexBug/hexdecode/buildpatterns.py

```diff
@@ -204,7 +204,18 @@ async def build_registry(session: ClientSession) -> Registry | None:
 
             with open(filename, "r", encoding="utf-8") as file:
                 for match in mod_info.registry_regex.finditer(file.read()):
-                    (pattern, direction, name, classname, is_great) = match.groups()
+                    named_groups = match.groupdict()
+                    if len(named_groups) > 0:
```

Comment:
I'd probably do this instead:
```suggestion
                    if named_groups := match.groupdict():
```
(though with how bad the rest of the codebase is, i'm not sure why i'm reviewing the code style tbh)